### ISSUE TITLE
Create QC to limit rare disease to the human disease branch

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6416]
rare diseases should be limited to the human disease branch. 

This Sparql query can be used: https://api.triplydb.com/s/CQyBQn-dk

**Comment by sabrinatoro** [comment]
There are 20 diseases that are 'rare' and are in the susceptibility branch. I will update them. 
(@matentzn : should I wait and keep these as a test for the QC?)

**Comment by matentzn** [comment]
You should be able to do this yourself! Just add the query as

src/sparql/qc/mondo/qc-susceptibility-rare.sparql

open a PR like https://github.com/monarch-initiative/mondo/pull/6248/files

and ask me for a review!

**Comment by sabrinatoro** [comment]
Let me ask the question differently: 
If I leave these errors (the 20 susceptibility terms that are in the rare subset), will it be caught in the PR? 
Or will it be caught only after the PR is merged (and therefore through off a failure for everything as soon as the PR is merge)?
(I think number 2, but I am hoping for number 1)

**Comment by sabrinatoro** [comment]
(and yes :-) I was planning of doing it myself :-) )

**Comment by matentzn** [comment]
I rcommend to write the failing check first, then fix. Two Different PRs!

So:
1. Make PR (PR1) that fails.
2. Fix everything on another PR. Nothing will fail (because the test is still on PR1). 
3. Merge master into PR1
4. If passes, merge.

**Comment by sabrinatoro** [comment]
Makes sense. Thank you!

**Comment by sabrinatoro** [comment]
Issue blocked by #6419
